src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -1944,10 +2013,31 @@ static void window_tile_inspector_paint(rct_window* w, rct_drawpixelinfo* dpi)
                 screenCoords.y = w->windowPos.y + w->widgets[WIDX_TRACK_SPINNER_HEIGHT].top;
                 gfx_draw_string_left(dpi, STR_TILE_INSPECTOR_BASE_HEIGHT_FULL, nullptr, COLOUR_WHITE, screenCoords);
 
+                // brakes/booster speed label
+                if (tileElement->AsTrack())
+                {
+                    if (tileElement->AsTrack()->GetTrackType() == TRACK_ELEM_BRAKES
+                        || tileElement->AsTrack()->GetTrackType() == TRACK_ELEM_BOOSTER)
+                    {
+                        screenCoords.y = w->windowPos.y + w->widgets[WIDX_TRACK_SPINNER_SPEED].top;
+                        gfx_draw_string_left(dpi, STR_TILE_INSPECTOR_TRACK_SPEED, nullptr, COLOUR_WHITE, screenCoords);
+
+                        // brakes/booster speed spinner value
+                        screenCoords.x = w->windowPos.x + w->widgets[WIDX_TRACK_SPINNER_SPEED].left + 3;
+                        screenCoords.y = w->windowPos.y + w->widgets[WIDX_TRACK_SPINNER_SPEED].top;
+                        int32_t speed = static_cast<int32_t>(tileElement->AsTrack()->GetBrakeBoosterSpeed());
+
+                        // convert the speed to a readable format
+                        gfx_draw_string_left(dpi, STR_FORMAT_INTEGER, &speed, COLOUR_WHITE, screenCoords);
```

Comment:
always use a Formatter instead of passing the value directly as we are phasing out the void* pointer versions of these functions.

Reply:
Also, `gfx_draw_string_left()` is deprecated. Use `DrawTextBasic()` instead.